cortex/export/save_views.py

```diff
@@ -99,7 +100,14 @@ def save_3d_views(
         else:
             view_name, view_params = view
         if isinstance(surface, str):
-            surface_params = unfold_view_params[surface]
+            surface_params = unfold_view_params[surface].copy()
+            # Fix unfold parameters if this subject doesn't have a flatmap
+            # Without a flatmap, the inflated surf corresponds to an unfold value of 1
+            # With a flatmap, the inflated surf corresponds to an unfold value of 0.5
+            if not has_flatmap:
+                surface_params["surface.{subject}.unfold"] = min(
+                    surface_params["surface.{subject}.unfold"] + 0.5, 1
```

Comment:
This transform maps the interval [0, 0.5] to [0.5, 1].
Wouldn't it be better to map to [0, 1]?
Such that:
 - `"fiducial"`: 0 ==> 0
 - `"inflated_less"`: 0.25 ==> 0.5
 - `"inflated"`: 0.5 ==> 1


```suggestion
                    surface_params["surface.{subject}.unfold"] * 2, 1
```

Reply:
Right. I'll blame COVID.